primeiro-pr-amem-leo

LeoHernandes opened the pull request:
Eu acredito

LeoHernandes committed:
agora vai

vickyad reviewed (commented):
Oi bem, então, olha só...
Seu swipe ta com probleminha de as vezes algumas peças de mesmo não estarem se juntando. Meu palpite é que tem a ver com a mudança do piece_index_1 e piece_index_2 em todas as instâncias (dentro de todos os if's), o que faz com que nem todos os casos sejam cobertos.

LeoHernandes commented:
> Oi bem, então, olha só...
> Seu swipe ta com probleminha de as vezes algumas peças de mesmo não estarem se juntando. Meu palpite é que tem a ver com a mudança do piece_index_1 e piece_index_2 em todas as instâncias (dentro de todos os if's), o que faz com que nem todos os casos sejam cobertos.

Ah sim sim, foi proposital. No site q eu fui as peças não empilhavam as somas

Por exemplo se numa linha tivesse:
8 - 4 - 2 - 2
E eu fizesse swipe left, só ia ficar:
8 - 4 - 4 - 0
Para a peça 4 somar com a outra teria q fazer mais um movimento.
Mas se a linha for
8 - 8 - 2 - 2
Daí com swipe left fica:
16 - 4 - 0 - 0
Pq tamo lidando com pares de peças diferentes. Foi nessa ideia q fui. Mas posso mudar lá

vickyad commented:
> > Oi bem, então, olha só...
> > Seu swipe ta com probleminha de as vezes algumas peças de mesmo não estarem se juntando. Meu palpite é que tem a ver com a mudança do piece_index_1 e piece_index_2 em todas as instâncias (dentro de todos os if's), o que faz com que nem todos os casos sejam cobertos.
> 
> Ah sim sim, foi proposital. No site q eu fui as peças não empilhavam as somas
> 
> Por exemplo se numa linha tivesse:
> 8 - 4 - 2 - 2
> E eu fizesse swipe left, só ia ficar:
> 8 - 4 - 4 - 0
> Para a peça 4 somar com a outra teria q fazer mais um movimento.
> Mas se a linha for
> 8 - 8 - 2 - 2
> Daí com swipe left fica:
> 16 - 4 - 0 - 0
> Pq tamo lidando com pares de peças diferentes. Foi nessa ideia q fui. Mas posso mudar lá

Não, é que não ta somando mesmo. Tipo
0 - 2  - 4 - 4 vai pra  2 - 0 - 4 - 4, por exemplo

LeoHernandes commented:
> > > Oi bem, então, olha só...
> > > Seu swipe ta com probleminha de as vezes algumas peças de mesmo não estarem se juntando. Meu palpite é que tem a ver com a mudança do piece_index_1 e piece_index_2 em todas as instâncias (dentro de todos os if's), o que faz com que nem todos os casos sejam cobertos.
> > 
> > 
> > Ah sim sim, foi proposital. No site q eu fui as peças não empilhavam as somas
> > Por exemplo se numa linha tivesse:
> > 8 - 4 - 2 - 2
> > E eu fizesse swipe left, só ia ficar:
> > 8 - 4 - 4 - 0
> > Para a peça 4 somar com a outra teria q fazer mais um movimento.
> > Mas se a linha for
> > 8 - 8 - 2 - 2
> > Daí com swipe left fica:
> > 16 - 4 - 0 - 0
> > Pq tamo lidando com pares de peças diferentes. Foi nessa ideia q fui. Mas posso mudar lá
> 
> Não, é que não ta somando mesmo. Tipo
> 0 - 2 - 4 - 4 vai pra 2 - 0 - 4 - 4, por exemplo

Nossa, desculpa. Ratiei, vou arrumar

LeoHernandes committed:
added a function to avoid the game crashing when trying to add a number to a non-empty board

LeoHernandes commented:
> > > Oi bem, então, olha só...
> > > Seu swipe ta com probleminha de as vezes algumas peças de mesmo não estarem se juntando. Meu palpite é que tem a ver com a mudança do piece_index_1 e piece_index_2 em todas as instâncias (dentro de todos os if's), o que faz com que nem todos os casos sejam cobertos.
> > 
> > 
> > Ah sim sim, foi proposital. No site q eu fui as peças não empilhavam as somas
> > Por exemplo se numa linha tivesse:
> > 8 - 4 - 2 - 2
> > E eu fizesse swipe left, só ia ficar:
> > 8 - 4 - 4 - 0
> > Para a peça 4 somar com a outra teria q fazer mais um movimento.
> > Mas se a linha for
> > 8 - 8 - 2 - 2
> > Daí com swipe left fica:
> > 16 - 4 - 0 - 0
> > Pq tamo lidando com pares de peças diferentes. Foi nessa ideia q fui. Mas posso mudar lá
> 
> Não, é que não ta somando mesmo. Tipo
> 0 - 2 - 4 - 4 vai pra 2 - 0 - 4 - 4, por exemplo
